frontend/src/components/ScheduledEventsTable/ScheduledEventsTable.tsx

```diff
@@ -1,29 +1,39 @@
-import React, { FunctionComponent, ReactNode } from 'react';
+import classNames from 'classnames';
+import React, { FunctionComponent, ReactNode, useState } from 'react';
 import { Table } from 'react-bootstrap';
 import { ScheduledEvent } from '../../types/scheduledEvents';
 import { ScheduledEventsTableRow } from '../ScheduledEventsTableRow';
-
 export interface ScheduledEventTableProps {
   currentPage: number;
   pageLimit: number;
   events: Array<{}>;
+  handleRunHistory: (id: number, name: string) => void;
 }
 export const ScheduledEventsTable: FunctionComponent<ScheduledEventTableProps> = (props) => {
-  const offset = (props.currentPage - 1) * props.pageLimit;
+  const [activeRow, setActiveRow] = useState(0);
+
+  const handleClick = (id: number, name: string) => (): void => {
```

Comment:
Glad to see you do this ... shows a lot better understanding of JS... Anyway, make this simpler ... just a function that takes in the event (or id & name), but shouldn't return a function... all that event info should be passed in from the `ScheduledEventsTableRow` component.